stories/webapp/peoplegroups/IndividualUser/IndividualUserActivity.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { string } from 'prop-types';
+import { Avatar } from 'lib';
+
+const pClasses = 'h-margin-clear h-margin-left-half';
+
+const IndividualUserActivity = ({ activeUser }) => (
+  <div className="h-margin-top">
+    <div className="h-display-flex h-margin-top-half h-margin-bottom-half">
```

Comment:
💚 Could create an array of activity data and map for each component. Probable quite a lot of work for not a lot of gain though.

Reply:
Yeah this is just kinda fudged in atm! I'm toying with the idea of making it its own little component, but this is already big enough :D